arbitrum-docs/partials/_gentle-intro-partial.mdx

```diff
@@ -57,23 +57,41 @@ Much development went into achieving this level of Ethereum compatibility. But a
+An <a data-quicklook-from="arbitrum-anytrust-chain">Arbitrum AnyTrust Chain</a> doesn’t have the same decentralization / trustlessness / permissionless security guarantees of a Rollup chain, and thus can offer lower fees. Rollup and AnyTrust are similar in many ways, though have one key difference: whereas in Rollup, all data is posted on L1 (which allows anyone to permissionless join as a validator), in AnyTrust, data is managed off-chain. In the case of a <a data-quicklook-from="challenge">Challenge</a>, an AnyTrust chain reverts back to “rollup mode”; the security assumption here is that at least 2 of the committee members are honest (i.e., they will provide the data when it’s necessary). Keeping the data off-chain in the happy/common case means the system can charge the user significantly lower fees.
 For applications that require high transaction throughput and don’t require the full decentralization that rollups provide, AnyTrust could be a sensible tradeoff.
 
 #### Q: So there's more than one Arbitrum chain out there?
 
-Yep! The fact that multiple chains can run in parallel is a crucial perk to off-chain scaling technology. Currently, on Ethereum mainnet, there are 2 Arbitrum chains: one Arbitrum Rollup chain, called ["Arbitrum One,"](https://portal.arbitrum.one/) and one AnyTrust chain, called ["Nova"](https://nova.arbitrum.io/); users and developers can pick whatever suits their security / transaction cost needs.
+Yep! The fact that multiple chains can run in parallel is a crucial perk to off-chain scaling technology.
 
-Developers also have the option of launching their own Arbitrum chains that run top an Arbitrum layer 2. These are called <a data-quicklook-from='arbitrum-orbit'>Orbit</a> chains and you can read more about them [here](/launch-orbit-chain/orbit-gentle-introduction.md).
+Here’s a snapshot of the chains running on Arbitrum:
+
+- On Ethereum mainnet:
+
+  - ["Arbitrum One"](https://portal.arbitrum.one/): an <a data-quicklook-from="arbitrum-rollup-chain">Arbitrum Rollup Chain</a> operated by the [Arbitrum Foundation](https://arbitrum.foundation)
+  - ["Nova"](https://nova.arbitrum.io/): an <a data-quicklook-from="arbitrum-anytrust-chain">AnyTrust Chain</a> operated by the [Arbitrum Foundation](https://arbitrum.foundation)
+
+- On Arbitrum One and Nova:
+  - Various L3 (Orbit) chain
+- On other L2s:
+  - Various L3 (Orbit) chain
+
+![700px-img](../assets/arbitrum-chains-diagram.jpeg)
+
+Developers can launch their own Arbitrum chains that run top an EVM L2 chain. These are called <a data-quicklook-from='arbitrum-orbit'>Orbit</a> chains and you can read more on our [Orbit introduction](/launch-orbit-chain/orbit-gentle-introduction.md).
```

Comment:
Developers can launch their own Arbitrum chains that run as an L2 chain on top of Ethereum.  They can also run Arbitrum chains that run on top of an EVM L2 chain.